src/components/ServerUtils.js

```diff
@@ -113,6 +113,7 @@ export class ServerRolesAccordion extends Component {
       let isOpen = (idx === this.state.accordionPosition);
       let valid = isRoleAssignmentValid(role);
       let triggerClass = valid ? '' : 'has-error';
+      let innerClass = this.props.mode && this.props.mode  === 'addserver' ? 'addserver': '';
```

Comment:
The initial check before the && is unnecesary. It would be more straightforward to use something like:
`   let innerClass = this.props.mode || '';
`

Reply:
@GarySmith  I was not sure if the mode will be eventually called addserver, just want to be flexible if there other mode.